### PR TITLE
Kennric/deploy

### PR DESCRIPTION
**Pull request by Kennric**
Merges various fixes necessary for deployment

## Changes in this PR.

- [X] Rubocop cleanups and mutes
- [X] Use strings in db collector rather than symbols for db name (compat with new settings system)
- [X] Travis conf fixes for testing db collector

## Testing this PR.

1. rake spec
2. rake rubocop
3. observe Travis test status

### Expected Output.

Tests pass.

@osuosl/devs

### CODE REVIEW

**Comment by Kennric** [comment]
WIP - still needs fix for db collector spec

**Comment by Kennric** [comment]
rubocop and spec fixed, please review @hsolorza @alxngyn @leian7

File: scheduler.rb [hunk 1]
```diff
@@ -3,49 +3,116 @@
...
+# schedule class will onlky start a schedule if one is not already running
+# TODO: make this simpler for Rubocop's sake
+class Scheduler
+  # rubocop:disable AbcSize, MethodLength
+  def self.start_unless_running(pid_file)
+    with_lockfile(File.join(File.dirname(pid_file), 'scheduler.lock')) do
+      if File.exist?(pid_file)
+        pid = IO.read(pid_file).to_i
+        if pid > 0 && process_running?(pid)
+          puts "not starting: scheduler is already is running with pid #{pid}"
```
Comment (hsolorza) [inline scheduler.rb]: there is a duplicate 'is'

**Comment by leian7** [comment]
+1

**Pull request merged**